### PR TITLE
Added documentation for [contenthash] #6839

### PR DESCRIPTION
**Pull request by FadySamirSadek**
Added output file name example for the new introduced [contenthash]

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -213,6 +214,12 @@ Using hashes based on each chunks' content:
 filename: "[chunkhash].bundle.js"
 ```
 
+Using hashes generated for extracted content :
```
Comment (montogeek) [inline src/content/configuration/output.md]: No space before colon

**Comment by FadySamirSadek** [comment]
@dhruvdutt @montogeek I applied the requested changes but there is an unrelated error in the scripts that is breaking the build and I believe it will do the same for all new PRs

File: src/content/configuration/output.md [hunk 2]
```diff
@@ -213,6 +214,12 @@ Using hashes based on each chunks' content:
 filename: "[chunkhash].bundle.js"
 ```
 
+Using hashes generated for extracted content:
```
Comment (edmorley) [inline src/content/configuration/output.md]: Is extracted content just content generated by eg `mini-css-extract-plugin` or does it include all other content (eg JS bundles) too? How does it differ from `chunkhash`?

Comment (FadySamirSadek) [reply src/content/configuration/output.md]: chunkhash is calculated for a chunk (an entry file) while content hash is a special hash generated by eg mini-css-extract-plugin and is calculated for extracted content only not the whole chunk content. This is very useful in the case of mini-css-extract-plugin as when an extracted css is changed the css hash only is invalidated not the entire chunk. 
did not want to add all these extra info here but will create a PR and add it with an example in https://webpack.js.org/guides/caching/#output-filenames  as per your issue

**Comment by montogeek** [comment]
@FadySamirSadek Can you add a note regarding https://github.com/webpack/webpack.js.org/issues/2096 ?

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**